src/wp-admin/security.php

```diff
@@ -0,0 +1,157 @@
+
+require_once( ABSPATH . 'wp-admin/admin-header.php' );
+
+$tabs = [
+    'users' => [
+        'title' => __('For Users'),
+        'classes' => ['nav-tab']
+    ],
+    'developers' => [
+        'title' => __('For Developers'),
+        'classes' => ['nav-tab']
+    ]
+];
+$active_tab = (in_array(@$_GET['tab'], array_keys($tabs)))
+    ? $_GET['tab']
+    : 'users';
+$tabs[$active_tab]['classes'][] = 'nav-tab-active';
+
+?>
+<div class="wrap">
+  <h1><?php echo esc_html( $title ); ?></h1>
+  <h2 class="nav-tab-wrapper">
+<?php   foreach ($tabs as $tab => $info): ?>
+    <a href="/wp-admin/security.php?tab=<?=$tab?>" class="<?=join(' ', $info['classes'])?>"><?=$info['title']?></a>
+<?php   endforeach; ?>
+  </h2>
+<?php
+switch ($active_tab):
+    case 'users':
+?>
+
+  <div class="card">
+    <h2><?php _e( '"Security First"' ); ?></h2>
+    <p><?php _e( "Security is important to business and it’s important to us here at ClassicPress. By bringing security forward to a place of greater prominence within the admin interface, we create a more streamlined experience for both users and developers." ); ?></p>
+    <p><?php _e( "As ClassicPress continues to evolve, the Security page will become the hub for all security features for ClassicPress core and 3<sup>rd</sup> party plugins that choose to support it." ); ?></p>
+    <p><?php printf(
+        /* translators: canonical link to security forum */
+        __( "Watch this page and the ClassicPress <a href=\"%s\" target=\"_blank\">Security forum</a> for more news as development continues." ),
+        'https://forums.classicpress.net/security'
+    ); ?></p>
+  </div>
+
+<?php
+        break;
+    case 'developers':
+?>
+
+  <div id="poststuff">
+    <div id="post-body" class="metabox-holder columns-2">
+      <div id="post-body-content">
+        <div class="meta-box-sortables ui-sortable">
+          <div class="postbox">
+            <h2><span><?php _e( 'Overview' ); ?></span></h2>
+            <div class="inside">
+              <div>
+                <p>The Security page is the first visible step in improving the overall approach to security in ClassicPress. Its pupose is to solve two related problems: <strong>discovery</strong> and <strong>auditing</strong>.</p>
+                <p>Most plugins are organised around their core purpose; after all, that purpose is the reason they were installed. Unfortunately this leads to poor discoverability of security-related settings - they may be in there somewhere, there may be none at all - without looking through everything there&lsquo;s no way to know.</p>
+                <p>Currenty, auditing the overall security profile of a ClassicPress site is impractical if there are more than a few plugins. Having all security-related settings from all plugins in one place means those settings can be autidted far more easily, as the time taken will be proportional to the number of settings, not the number of places to look for those settings.</p>
```

Comment:
This is a good, clear explanation of the security screen and the problems it is intended to solve.

- Typos in the last line: `Currenty`, `autidted`
- Text needs to be wrapped in `__( )` for translation

Reply:
Fixed, and yes, but that's the very last thing as it makes editing the blurb harder (imho).